index.html

```diff
@@ -226,150 +249,169 @@ <h4>Multi-Post Stories Gain+Glory</h4>
-            </fieldset>
-          </div>
+    <div class="bg_dsk2">
+      <section class="section-contact-form">
+        <div class="div-heading-form">
+          <h2 class="heading-form">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+          </h2>
+        </div>
 
-          <div class="div_field">
-            <fieldset>
-              <input class="filter" type="email" placeholder="Email address" required />
-            </fieldset>
-          </div>
+        <div class="div_form_container">
+          <form class="contact-form" action="https://formspree.io/f/xzbwnlee" method="post">
+            <div class="div_field">
+              <fieldset>
+                <input class="filter" type="text" placeholder="Fullname" maxlength="30" required />
+              </fieldset>
+            </div>
 
-          <div class="div_field">
-            <fieldset>
-              <textarea class="filter" name="comment" placeholder="Enter text here" maxlength="500" required></textarea>
-            </fieldset>
-          </div>
+            <div class="div_field">
+              <fieldset>
+                <input class="filter" type="email" placeholder="Email address" required />
+              </fieldset>
+            </div>
 
-          <div class="divbtn_field">
-            <button class="btn-form btn_effects" type="submit">Get in touch</button>
-          </div>
+            <div class="div_field">
+              <fieldset>
+                <textarea
+                  class="filter txtarea_mob"
+                  name="comment"
+                  placeholder="Enter text here"
+                  maxlength="500"
+                  required
+                ></textarea>
+                <textarea
+                  class="filter txtarea_dsk"
+                  name="comment"
+                  placeholder="Typing something"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </fieldset>
+            </div>
 
-          <ul class="social_links">
-            <li>
-              <a href="https://twitter.com/marcealarconb"> <img src="images/twitter.svg" /></a>
-            </li>
-            <li>
-              <a href="#" target="_blank"><img src="images/linkedin.svg" /></a>
-            </li>
-            <li>
-              <a href="#" target="_blank"> <img src="images/medium.svg" /></a>
-            </li>
-            <li>
-              <a href="#" target="_blank"> <img src="images/github.svg" /></a>
-            </li>
-            <li>
-              <a href="#" target="_blank"> <img src="images/angellist.svg" /></a>
-            </li>
-          </ul>
-        </form>
-      </div>
-    </section>
+            <div class="divbtn_field">
+              <button class="btn-form btn_effects" type="submit">Get in touch</button>
+            </div>
+
+            <hr>
```

Comment:
- [ ] please note that the `<hr>` that you added as requested by the previous reviewer is not visible and its in the wrong place , kindly fix it 

![image](https://user-images.githubusercontent.com/89282221/208125602-89d56c63-7ec4-45df-89b6-02036815b6e4.png)

Reply:
Dear @ichala, Could you give me your feedback if the hr tag is now in the requested position?
Thank you very much
![image](https://user-images.githubusercontent.com/65084923/208128693-6315069f-5a68-4c54-9565-afb25cfdbdf3.png)
![image](https://user-images.githubusercontent.com/65084923/208128822-92abe7e9-49b3-4abf-9edf-cb91e2f25608.png)

Reply:
It looks like you fixed the issue and ur project is approved 💯💯💯 Good job , keep rocking